### PR TITLE
Fixes analytics for WayFair

### PR DESCRIPTION
**Pull request by mike-lvov**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1641

### CODE REVIEW

**Review by colinhowe — APPROVED** [review]
Might as well delete the commented out code. Otherwise, looks great

File: functions/analytics.js [hunk 1]
```diff
@@ -122,11 +166,11 @@ exports.generateAnalytics = functions.https.onCall(async (data, context) => {
...
       const { email } = authUsersById[id] || {};
-      const visitIds = visits.map((el) => el.id);
+      // const visitIds = visits.map((el) => el.id);
 
-      visitIds.map((visit) =>
-        fs.writeFileSync(allSpaceVisitsFileName, `${visit} \n`, { flag: "a" })
-      );
+      // visitIds.map((visit) =>
+      // fs.writeFileSync(allSpaceVisitsFileName, `${visit} \n`, { flag: "a" })
+      // );
```
Comment (goran-peoski-work) [inline functions/analytics.js:173]: These kinds of comments tend to drift away with time. Might be easier to just delete the lines.

https://github.com/sparkletown/sparkle/blob/redesign-2021/docs/code-style.md#maybe-could-be-useful-later-dead-code

**Review by goran-peoski-work — APPROVED** [review]
LGTM

**Comment by goran-peoski-work** [comment]
> Might as well delete the commented out code. Otherwise, looks great

Wasn't aware of this review when I made this https://github.com/sparkletown/sparkle/pull/2909#discussion_r823557840 😄

**Pull request merged**